deploy/servctl_utils/shell_utils.py

```diff
@@ -31,3 +33,79 @@ def wait_for(procs):
 
     for proc in procs:
         proc.wait()
+
+
+def run(command,
```

Comment:
this is directly copy/pasted from `seqr/utils/shell_utils.py` so I wouldn't worry about reviewing it